package/Dockerfile.windows

```diff
@@ -106,7 +106,7 @@ FROM mcr.microsoft.com/powershell:nanoserver-${SERVERCORE_VERSION}
 USER ContainerAdministrator
 COPY --from=builder /Windows/System32/netapi32.dll /Windows/System32/
 COPY --from=builder /wins.exe /confd.exe /Windows/
-COPY --from=builder /nginx-1.15.9 /etc/nginx
+COPY --from=builder /nginx-1.19.9 /etc/nginx
```

Comment:
this is necessary? i have no context

Reply:
We are packaging a 2.5 year old version of nginx currently, which doesn't sit well with me 
```
nginx-1.15.9.tar.gz                                26-Feb-2019 15:38             1031760
```